sos/report/plugins/logs.py

```diff
@@ -56,6 +56,7 @@ def setup(self):
 
         self.add_cmd_output("journalctl --disk-usage")
         self.add_cmd_output('ls -alRh /var/log/')
+        self.add_cmd_output('ls -la /usr/lib/snapper/')
```

Comment:
Why excatly `logs` plugin shall list directory for a user libraries of a system recovery and snapshot management tool?

Reply:
Please let us know in which plugin we can collect this data..

Reply:
It looks like a new `snapper` plugin would be appropriate to capture this as well as other `snapper` relevant config files and/or command output.